sdk/storage/storage-internal-avro/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@azure/storage-internal-avro",
+  "sideEffect": false,
+  "private": true,
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-preview.1",
+  "description": "internal avro parser",
+  "license": "MIT",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "main": "./srt/index.ts",
+  "module": "dist-esm/index.js",
+  "types": "./types/index.d.ts",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "build:es6": "tsc -p tsconfig.json",
+    "build:nodebrowser": "rollup -c 2>&1",
```

Comment:
What does `build:nodebrowser` do?

Reply:
It bundles all the individual .js file into a single index.js. Copied the script from storage-blob.

Reply:
It's useless here. Please cleanup unnecessary entries in package.json.